simulation/state.go

```diff
@@ -36,6 +34,9 @@ const (
 	InitiallyBondedValidators = "initially_bonded_validators"
 )
 
+// updateBankState updates the bank genesis state.
```

Comment:
The update naming seems off in this file, we return a new genesis variable instead of update a variable, should rather be named
```
bankGenesisState
evmGenesisState
...
```
I would also rename the file into genesis.go

Reply:
We extract the module specific portion of the genesis from the rawstate, update and return it. The default values are still sustained . I do think update matches the logic more in this case .

The main function for this file is the `AppStateFn` which , basically returns the application state and is used by `SimulateFromSeed` function by the sdk .